src/apps/dashboard/routes/libraries/index.tsx

```diff
@@ -0,0 +1,115 @@
+    const startTask = useStartTask();
+    const { data: tasks, isPending: isLiveTasksPending } = useLiveTasks({ isHidden: false });
+
+    const gridLayout = useMemo(() => ({
+        xs: 12,
+        sm: 6,
+        md: 3,
+        lg: 2.4
+    }), []);
+
+    const librariesTask = useMemo(() => (
+        tasks?.find((value) => value.Key === 'RefreshLibrary')
+    ), [ tasks ]);
+
+    const showMediaLibraryCreator = useCallback(() => {
+        const mediaLibraryCreator = new MediaLibraryCreator({
+            collectionTypeOptions: getCollectionTypeOptions()
+        }) as Promise<boolean>;
+
+        void mediaLibraryCreator.then((hasChanges: boolean) => {
+            if (hasChanges) {
+                void queryClient.invalidateQueries({
+                    queryKey: ['VirtualFolders']
+                });
+            }
+        });
+    }, []);
+
+    const onScanLibraries = useCallback(() => {
+        if (librariesTask?.Id) {
+            startTask.mutate({
+                taskId: librariesTask.Id
+            });
+        }
+    }, [ startTask, librariesTask ]);
+
+    if (isVirtualFoldersPending || isLiveTasksPending) return <Loading />;
+
+    return (
+        <Page
+            id='mediaLibraryPage'
+            title={globalize.translate('HeaderLibraries')}
+            className='mainAnimatedPage type-interior'
+        >
+            <Box className='content-primary'>
+                <Stack spacing={3} mt={2}>
+                    <Stack direction='row' alignItems={'center'} spacing={2}>
+                        <Button
+                            onClick={onScanLibraries}
+                            startIcon={<RefreshIcon />}
+                            sx={{
+                                alignSelf: 'flex-start',
+                                fontWeight: 'bold'
+                            }}
+                        >
+                            {globalize.translate('ButtonScanAllLibraries')}
+                        </Button>
+                        {(librariesTask && librariesTask.State == TaskState.Running) && (
+                            <TaskProgress task={librariesTask} />
+                        )}
+                    </Stack>
+
+                    <Box>
+                        <Grid container spacing={2}>
+                            {virtualFolders?.map(virtualFolder => (
+                                <Grid
+                                    key={virtualFolder?.ItemId}
+                                    item
+                                    { ...gridLayout }
+                                >
+                                    <LibraryCard
+                                        virtualFolder={virtualFolder}
+                                    />
+                                </Grid>
+                            ))}
+
+                            <Grid item { ...gridLayout }>
+                                <BaseCard
```

Comment:
Instead of using a card for this, let's add it as a button at the top of the screen before the scan button.